server/app/crud/crud.py

```diff
@@ -11,10 +11,17 @@ def create_table(db_file):
     connection = create_connection(db_file)
     cursor = connection.cursor()
     cursor.execute("""
+    CREATE TABLE IF NOT EXISTS generated_keys (
```

Comment:
I just realised we do not need to create a seperate table for storing the key. Because every write request is logged in the store table in sqllite3 which means we can initialise the counter from the last value from the store table.